### DISCUSSION TITLE
Problem with Parsing Timestamp in struct

### DISCUSSION THREAD

**Discussion by rolson-alister** [sijms/go-ora#185] [Q&A] [answered]
Hey!

I was tormented all evening, but could not solve the problem. I hope for your help!

The crux of the problem is this:

I have the following table in the database:
![image](https://user-images.githubusercontent.com/68353547/162824711-04e07a95-98f4-4e91-a754-176503f2c97a.png)

for this table created the following structure:
![image](https://user-images.githubusercontent.com/68353547/162824843-5d5ce27b-8962-40c6-a69c-84b8e3eeb805.png)

I run the following code to get the data:
![image](https://user-images.githubusercontent.com/68353547/162825039-69786839-fab4-4180-af40-f82466693820.png)

but at runtime I get the following error:
![image](https://user-images.githubusercontent.com/68353547/162825132-9668321f-bff3-4e27-81ac-7ada61404a80.png)

but if I change the type for the ExpiresAt field to string in the structure, everything works without errors. But I need exactly the type time.Time

Can you please tell me what I'm doing wrong? Thank you!

I hope for your help.

**Comment by sijms** [comment]
sorry for late 
ok I will test it and see the problem

**Reply by rolson-alister** [reply]
Hello!
Thank you for helping.
I will be waiting for your feedback.

**Comment by sijms** [comment, marked as answer]
I test you code the problem when you pass null in field 5 or 6 (timestamp) it will not fit
thus you need to change field 5 and 6 in your Client struct into sql.NullTime

also you will face same problem in all nullable fields e.g. Description you need to use sql.NullString

**Reply by rolson-alister** [reply]
Oh, thank you so much for the information. 
Everything worked!

Thanks again so much for your help!
